core/primitives/src/runtime/config.rs

```diff
@@ -24,6 +26,13 @@ pub struct RuntimeConfig {
     pub wasm_config: VMConfig,
     /// Config that defines rules for account creation.
     pub account_creation_config: AccountCreationConfig,
+    /// Storage usage delta that need to be added on migration of account from v1 to v2
+    /// See https://github.com/near/nearcore/issues/3824
+    /// Account list for mainnet generated by utils/storage-usage-delta-calculator
+    #[serde(default = "RuntimeConfig::empty_storage_usage_delta")]
+    #[serde(skip)]
+    #[cfg(feature = "protocol_feature_add_account_versions")]
+    pub storage_usage_delta: &'static HashMap<AccountId, u64>,
```

Comment:
What is the reason for doing this? Couldn't we have it as a global variable? I believe `&'static` should be used with caution

Reply:
Yes, global variable seems to be better suited